### PR TITLE
feat: add sample.env and .gitignore files

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+PORT=
+DB_URI=
```
Comment (naomi-lgbt) [inline sample.env]: https://github.com/freeCodeCamp/boilerplate-project-library/blob/7692c6ab272e1f87cbaf3c7df89e5dbd89de7afb/routes/api.js#L14

We've named this one differently in @SaintPeter's pull request #5 - should we update that PR to reflect this variable name?

Comment (scissorsneedfoodtoo) [reply sample.env]: Hi @nhcarrigan, thanks for catching that. I just noticed this myself.

Either is totally fine, and learners can change it themselves later if they want.

I'll go ahead and change it here to reflect what's in PR #5.

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳

**Comment by scissorsneedfoodtoo** [comment]
Thanks again for taking a look at this @nhcarrigan, and for all of your help with these boilerplates, along with everything else you've been doing :slightly_smiling_face:

**Comment by naomi-lgbt** [comment]
Of course! Always happy to help! 😁

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Comment by ShaunSHamilton** [comment]
Does #5  need to be merged before this for any reason?

**Comment by naomi-lgbt** [comment]
@Sky020 I don't see any overlap so this should be good to merge whenever. 🙂

**Pull request merged**